.travis.yml

```diff
@@ -163,7 +164,6 @@ jobs:
         - |
           # Run integration tests.
           version="$(./linkerd version --client --short | tr -cd '[:alnum:]-')"
-          ./bin/test-cleanup
```

Comment:
This is now done in the `bin/test-run` script so that it can be shared between `staging` and `l5d-bot`.